src/main/resources/portal.properties.EXAMPLE

```diff
@@ -165,7 +165,7 @@ segfile.url=http://cbio.mskcc.org/cancergenomics/gdac-portal/seg/
 show.oncokb=true
```

Comment:
Is it possible to specify the API URL?

Reply:
Please see below. `oncokb.public_api.url` can be used